device-types/Extreme Networks/summit-x460-48p.yml

```diff
@@ -0,0 +1,333 @@
+---
+manufacturer: Extreme Networks
+model: Summit X460-48p
+slug: extreme-networks-summit-x460-48p
+part_number: '16404'
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```

Comment:
Why is this here? This definition is for `device-bays` where a device has a child that slots into it, or itself is a child that slots into another device. If this is not the case, please remove this definition

Reply:
I've removed **subdevice_role: parent** and **is_full_depth: true**, for the rest, it's useful (I think).